iac-common/src/testFixtures/java/org/sonar/iac/common/testing/Verifier.java

```diff
@@ -244,22 +250,44 @@ public void reportIssue(HasTextRange toHighlight, String message, List<Secondary
     }
 
     protected void reportIssue(TextRange textRange, String message, List<SecondaryLocation> secondaryLocations) {
-      Issue issue = new Issue(textRange, message, secondaryLocations);
+      var issue = new Issue(textRange, message, secondaryLocations);
       if (!raisedIssues.contains(issue)) {
         TextPointer start = textRange.start();
         TextPointer end = textRange.end();
-        SingleFileVerifier.Issue reportedIssue = verifier
+
+        var reportedIssue = ((SingleFileVerifier) verifier)
```

Comment:
Yes, because otherwise we would get `InternalIssue` back, which we can't use.
This way we get `SingleFileVerifier.IssueBuilder` back, which we are allowed to use.

Reply:
I decided to change InternalIssueVerifier to MultiFileVerifier, as it's more readable and understandable.
Also I decided to add some commentary to explain this.